app/src/main/java/uk/gov/onelogin/login/biooptin/BiometricPreferenceModule.kt

```diff
@@ -3,14 +3,14 @@ package uk.gov.onelogin.login.biooptin
 import dagger.Binds
 import dagger.Module
 import dagger.hilt.InstallIn
-import dagger.hilt.android.components.ViewModelComponent
-import dagger.hilt.android.scopes.ViewModelScoped
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
 
 @Module
-@InstallIn(ViewModelComponent::class)
+@InstallIn(SingletonComponent::class)
```

Comment:
Any reason for this change?

Reply:
SecureStore is linked to the App lifecycle/Singleton/Global. 
SecureStore depends on BiometricPreferenceHandler so it can fetch the preference and initialise. Therefore I had to open the scope of this to Singleton, so it can be injected (and used) when the app starts (rather than the limited scope of the BiometricViewModel)